### PR TITLE
Allow hiding of the icon title.

### PR DESCRIPTION
**Pull request by Cwiiis**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -612,7 +630,7 @@ window.GaiaAppIcon = (function(exports) {
   template.innerHTML =
     `<style>@import url(${stylesheet});</style>
      <div id='image-container'><div id="spinner"></div></div>
-     <div><div dir='auto' id='subtitle'></div></div>`;
+     <div id='title-container'><div dir='auto' id='subtitle'></div></div>`;
```
Comment (gmarty) [inline script.js]: nit: can we make sure we always use `"` in HTML attributes, even when quoted in JS strings?

File: script.js [hunk 2]
```diff
@@ -249,6 +251,22 @@ window.GaiaAppIcon = (function(exports) {
...
+  Object.defineProperty(proto, 'showName', {
+    get: function() {
+      return this._showName;
+    },
+
+    set: function(show) {
+      show = show ? true : false;
+      if (this._showName !== show) {
+        this._showName = show;
+        this._titleContainer.classList.toggle('hidden', !show);
```
Comment (gmarty) [inline script.js:263]: You're not using the `hidden` property here so that you can animate opacity, right?

File: script.js (hunk 1 above)
Comment (Cwiiis) [inline script.js]: Will do.

File: script.js (hunk 2 above)
Comment (Cwiiis) [inline script.js:263]: We discussed this on IRC. Short answer is 'yes', long answer is 'I forgot about the hidden property, but yes' :)

**Pull request merged**